mlflow/utils/databricks_utils.py

```diff
@@ -162,6 +163,20 @@ def get_databricks_host_creds(profile=None):
         config = provider.ProfileConfigProvider(profile).get_config()
     else:
         config = provider.get_config()
+    # if a path is specified, that implies a Databricks tracking URI of the form:
+    # databricks://profile-name/path-specifier
+    if (not config or not config.host) and path:
+        dbutils = _get_dbutils()
+        # Prefix differentiates users and is provided as path information in the URI
+        key_prefix = path
```

Comment:
nit: this can go inside `if dbutils:` right?

Reply:
Yup - fixed